tests/README.md

```diff
@@ -0,0 +1,40 @@
+## Run app locally in e2e setup
+
+Currently tests are running against http://localhost:3000 (If you need to change the baseURL you need to do it in the ./cypress.sh), so the app needs to be running. Either follow the e2e setup in [next app readme](../next/README.md), or rebuild & restart via the following npm script from this directory:
+
+```
+yarn start:e2e
+```
+
+## Run Cypress locally
+
+```
+npm run cypress:open //for open mode (Running tests in chrome browser)
```

Comment:
Do we use `yarn` or `npm`?

Reply:
The use of both is not exclusive, but I will rewrite it so that only yarn will be used everywhere.

Reply:
fixed